lib/helper-functions.php

```diff
@@ -371,7 +371,7 @@ function rscandir($base = '', &$data = [])
  *
  * @phpstan-param non-empty-string $delimiter
  */
-function explode_and_trim($delimiter, $string = '', $limit = 0, $useArrayFilter = true)
+function explode_and_trim($delimiter, $string = '', $limit = PHP_INT_MAX, $useArrayFilter = true)
```

Comment:
Maybe we should remove the default parameter for `$string`, too. It doesn't really make sense to call this with an empty string, does it?

```suggestion
function explode_and_trim($delimiter, $string, $limit = PHP_INT_MAX, $useArrayFilter = true)
```